src/Liquid/StandardFilters.php

```diff
@@ -43,7 +43,8 @@ public static function append($input, $string)
 	public static function capitalize($input)
 	{
 		return preg_replace_callback("/(^|[^\p{L}'])([\p{Ll}])/u", function ($matches) {
-			return $matches[1] . ucfirst($matches[2]);
+			$first_char = mb_substr($matches[2], 0, 1);
+			return $matches[1] . mb_strtoupper($first_char) . mb_substr($matches[2], 1, null);
```

Comment:
```suggestion
			return $matches[1] . mb_strtoupper($first_char) . mb_substr($matches[2], 1);
```
I had to check the manual to see if `null` has a special meaning here. Therefore I'd say this reads much faster without extra null.